plugins/node/instrumentation-dataloader/src/instrumentation.ts

```diff
@@ -89,40 +92,53 @@ export class DataloaderInstrumentation extends InstrumentationBase {
     return hasParentSpan || !config.requireParentSpan;
   }
 
+  private getSpanName(
+    dataloader: DataloaderInternal,
+    operation: 'load' | 'loadMany' | 'batch'
+  ): string {
+    const dataloaderName = dataloader.name;
+    if (dataloaderName === undefined || dataloaderName === null) {
```

Comment:
This can be written as shortcut 
```suggestion
    if (dataloaderName == null) {
```

or even
```suggestion
    if (!dataloaderName) {
```
to also catch empty strings